lab_7_llm/assets/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
Remove all files for lab 7 from the PR. If you want to save them, move them to a separate branch